samples/data-sealing/CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+cmake_minimum_required(VERSION 3.10)
+
+project("Data Sealing Sample" LANGUAGES C CXX)
+
+find_package(Threads REQUIRED)
+
+find_package(OpenEnclave CONFIG REQUIRED)
+
+# TODO: Remove this when the package exports the target.
+add_executable(openenclave::oeedger8r IMPORTED)
```

Comment:
I've heard on IRC that we can move this into our CMake package by manually injecting some stuff into it, which would be nice. It's also in the PR checklist, but I think we can do it later.

Reply:
Moved to #1539.